Gems/ROS2/Code/Source/Camera/CameraSensor.cpp

```diff
@@ -71,6 +71,8 @@ namespace ROS2
         AZ_Assert(
             m_cameraConfiguration.m_verticalFieldOfViewDeg > 0.0f && m_cameraConfiguration.m_verticalFieldOfViewDeg < 180.0f,
             "Vertical fov should be in range 0.0 < FoV < 180.0 degrees");
+        // AZ_Assert(m_cameraConfiguration.m_horizontalFieldOfViewDeg > 0.0f && m_cameraConfiguration.m_horizontalFieldOfViewDeg < 360.0f,
+        //     "Horizontal fov should be in range 0.0 < FoV < 360.0 degrees");
```

Comment:
remove comment. Also, These fields should be limited in the Reflect instead, that might be a good addition within this PR to add such limits on input. Better to limit than to assert.

Reply:
Sorry, I haven't spotted the comment. Will remove and modify to add limits on params.